lib/routes/account.js

```diff
@@ -1017,10 +1092,20 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenWithVerificationStatus(sessionToken.id)
```

Comment:
It's not obvious to me why we're looking up the token again here...couldn't we use the `sessionTokenWithVerificationStatus` authentication scheme and get this extra data slurped in when we check the token the first time?